src/ViewModel/Builder/PictureBuilder.php

```diff
@@ -109,8 +109,9 @@ public function build() : ViewModel\Picture
                     }
 
                     $uri = call_user_func($this->uriGenerator, $type, $width, $height);
+                    $scalingFactor = round($width / $size['width'], 1);
```

Comment:
I might rethink this area, ideally we want to try and produce a few sizes (3x, 2.5x, 2x, 1.5x, 1x).

Reply:
We could provide the array of scales currently hard coded at https://github.com/elifesciences/journal/blob/develop/src/ViewModel/Builder/PictureBuilder.php#L114 with an optional parameter to the PictureBuilder constructor (and then via an object property), with `[2, 1]` being the default. Does that work for what you have in mind?

Reply:
With fb592695b470a53bd0966b7386770d7bf2ddb7f7 it can now safely produce any size that we want, but I'll leave this for now.